api/src/org/labkey/api/exp/api/ExperimentService.java

```diff
@@ -470,10 +472,14 @@ ExpMaterialProtocolInput createMaterialProtocolInput(
 
     String generateGuidLSID(Container container, Class<? extends ExpObject> clazz);
 
+    Pair<String, String> generateLsidWithDBSeq(@NotNull Container container, Class<? extends ExpObject> clazz);
```

Comment:
```suggestion
    Pair<String, String> generateLSIDWithDBSeq(@NotNull Container container, Class<? extends ExpObject> clazz);
```
Just to be consistent with the other `generateLSID` names.